hw2/style-hw2.css

```diff
@@ -0,0 +1,214 @@
+* {
+	padding: 0;
+	margin: 0;
+}
+
+body {
+	background-color: #f9f9f9;
+}
+/*----------header----------*/
+.container {
+	width: 75%;
+	background-color: #fff;
+	margin: 0 auto;
+	box-shadow: 0 0 3px 2px #dbdbdb;
+	padding: 50px 45px 60px 45px;
+}
+
+.main-menu {
+	margin-top: 35px;
+	height: 40px;
+	width: 100%;
+	background: linear-gradient(to top, #444343, #5e5e5e);
+}
+
+.main-menu__item {
+	list-style: none;
+	float: left; 
+	border-right: rgba(255,255,255,0) 1px solid;
+	border-left: rgba(255,255,255,0) 1px solid;
+}
+
+.main-menu__link {
+	color: #ffffff;
+	text-decoration: none;
+	font-family: sans-serif;
+	display: block;
+	height: 100%;
+	width: 100px;
+	padding: 10px 25px;
+} 
+
+.main-menu__item:hover {
+	border-right: rgba(255,255,255,1) 1px solid;
+	border-left: rgba(255,255,255,1) 1px solid;
+	background: linear-gradient(to top, #2f2f2f, #4a4a4a);
+}
+/*----------aside----------*/
+.products {
+	margin-top: 40px;
+	margin-right: 40px;
+	width: 25%;
+	display: inline-block;
+	border: 1px solid #c8c8c8;
+}
+
+
+.products__item {
+	list-style: none;
+	border-bottom: 1px solid #c8c8c8;
+	color: #807e7e;
+}
+
+.products__item:last-child {
+	border-bottom: 0px solid #c8c8c8;
+}
+
+.products__link{
+	color: #807e7e;
+	text-decoration: none;
+	font-family: sans-serif;
+	display: block;
+	height: 100%;
+	padding: 15px 25px;
+} 
+
+.products__item:hover {
+	background-color: #f9f9f9;
+} 
+
+.products__item:hover .products__link {
+	color: #000;
+	padding-left: 35px;
+} 
+/*----------main----------*/
+.main-info {
+	width: 69%;
+	margin-top: 40px;
+	vertical-align: top;
+	overflow: hidden;
```

Comment:
in menu I prefer to use clearfix, as far here could be dropdown that will be cut by this overflow